### PR TITLE
Event finished view

### PR DESCRIPTION
**Pull request by Andreslw**
New event finished button added with notification.

New notification context created to call a notification at  n level of the tree

### CODE REVIEW

File: src/partner/App.tsx [hunk 1]
```diff
@@ -12,17 +13,19 @@ interface IPartnerAppProps {
...
-          path={`${props.match.url}/current-events`}
-          component={CurrentEventsView}
-        />
-        <Route
-          path={`${props.match.url}/past-events`}
-          component={PastEventsView}
-        />
-        <Route path={`${props.match.url}/events/:id`} component={EventView} />
-      </Switch>
+      <NotificationContextProvider.NotificationContextProvider>
```
Comment (mishelashala) [inline src/partner/App.tsx]: Shouldn't this be `NotificationContextProvider.Provider`?

File: src/partner/modules/EventList/EventListItem.tsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 import { Link } from "react-router-dom";
-import { ListStyled } from "src/partner/modules/ui";
 import { IEvent } from "src/partner/models/Event";
 import { Modal } from "src/partner/modules/ui/Modal/Modal";
 import { Notification } from "src/partner/modules/ui/Modal/Notification";
 import { CreateEvent } from "src/components/event/Create";
 import styledComponents from "styled-components";
+import {
+  ListStyled,
+  NotificationContextProvider
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: `NotificationContextProvider` should be in `providers/NotificationContextProvider`

**Review by mishelashala — CHANGES_REQUESTED** [review]

File: src/partner/App.tsx (hunk 1 above)
Comment (Andreslw) [reply src/partner/App.tsx]: It's already used in the NotificationProvider (renamed) component inside providers/NotificationContext.tsx

File: src/providers/NotificationContext.tsx [hunk 3]
```diff
@@ -8,12 +8,14 @@ export const NotificationContext = React.createContext({
   handleCloseNotification: () => {}
 });
 
+export const NotificationConsumer = NotificationContext.Consumer;
```
Comment (mishelashala) [inline src/providers/NotificationContext.tsx]: don't export manually the consumer

**Review by mishelashala — CHANGES_REQUESTED** [review]

**Review by mishelashala — APPROVED** [review]

**Pull request merged**